foundryup/foundryup

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
 set -e
 
-FOUNDRY_DIR=${FOUNDRY_DIR-"$HOME/.foundry"}
+BASE_DIR=${XDG_CONFIG_HOME:-$HOME}
+FOUNDRY_DIR=${FOUNDRY_DIR-"$BASE_DIR/.foundry"}
```

Comment:
I don't really have strong opinions about xdg paths,
but I know this has been requested a few times.
Using the basedir as fallback is reasonable imo.

I assume if the xdg paths are set then the user knows what they're doing.

tested locally works as expected without xdg paths set